docs/installation_and_tutorial.md

```diff
@@ -408,4 +354,4 @@ You can also activate the trace forwarder and receive the traces with Trace View
 C:\ProgramFiles\Siemens\Automation\WinCCUnified\bin\RTILtraceTool.exe -mode receiver -tcp -host <IP of Edgebox> -port 35505
```

Comment:
I would also add a screenshot which shows the TraceViewer and the command prompt on Windows side to activate the trace viewer as receiver and show the traces